cargo/cargo_build_script_runner/lib.rs

```diff
@@ -121,7 +121,7 @@ impl BuildScriptOutput {
                 }
             })
             .collect::<Vec<_>>()
-            .join(" ")
+            .join("\n")
```

Comment:
No, it's because it's easier to read new line separated file in C++, the stdlib doesn't have a lot of facilities for splitting using spaces... So the wrapper expects this.

Reply:
Ok, make sense :)